WordPress/src/main/res/values/colors.xml

```diff
@@ -325,5 +325,7 @@
 
     <!--Reader Post Card -->
     <color name="reader_post_list_background">@color/neutral_0</color>
+    <color name="reader_welcome_card_background">@color/primary_emphasis_lowest</color>
+    <color name="reader_welcome_card_text">@color/blue_70</color>
```

Comment:
Had to add these colors in `colors.xml` as I couldn't find any existing palette color that could be used on both light and dark modes for welcome card text and background.

Reply:
iirc we had to change the empty slate view to match colors within the palette. I wonder if we need to adjust here as well, but that's more of a question for @mbshakti

Reply:
Will check with Shakti and update.

Reply:
Yep, I agree that we might want to update the colors in the design instead of introducing a new set of colors.

Reply:
I discussed with Shakti, quoting her:
>  it’s such a custom/rare card that we probably won’t reuse else where

With this limitation, I think we can keep these colors here or introduce new colors 
`colorReaderWelcomeCardBackground`
`colorReaderWelcomeCardText`
in the `values/style.xml` and `values-night/style.xml` similar to `colorGutenbergBackground`

Let me know what you think.

Reply:
If I understand our approach for the colors.xml correctly this addition is still not following it.

I believe it's ok-ish to add `reader_welcome_card_background` and `reader_welcome_card_text`, since these two entries need to have custom colors in light vs dark mode. However, I'm wondering if `reader_welcome_green_dot`, `reader_welcome_purple_dot`, ...., should be inlined directly into the drawable ?!?. 

P.S. Ideally, we wouldn't be using the colors(green, purple, yellow) directly, but we'd be using semantic colors - however, I believe it's fine to use the ["STUDIO 2.2.0"](https://github.com/wordpress-mobile/WordPress-Android/blob/d7146bac7f9ee4cc006491c6ed9b5b7818bf41a0/WordPress/src/main/res/values/colors.xml#L148) directly for this usecase as it's an illustration.

Reply:
> I believe it's fine to use the "STUDIO 2.2.0" directly for this usecase as it's an illustration.

Used them directly here: cc99e75

Reply:
> Are you talking about reader_following_empty_view_background color? I see it is added to values/colors.xml, values-night/colors.xml and not picked from style color attributes in values/style.xml and values-night/style.xml.

Yep, for that they needed the custom colors for light/dark, but the others were changed to palette. I believe the changes you made are good to go. 👍